docs/model_zoo/action.rst

```diff
@@ -0,0 +1,62 @@
+.. _gluoncv-model-zoo-action:
+
+Action
```

Comment:
Is "Action" a good name without confusion?

Reply:
Changed to Action Recognition.